### PR TITLE
Refactor/unified shares

### PR DESCRIPTION
**Pull request by tcrosado**
Refactor in order to have share percentages defined in a single library.

Before opening a pull request, please ensure:

- [x] You have followed our [**contributing guidelines**](https://github.com/AppStoreFoundation/asf-contracts/wiki/Contributing-to-AppCoins-Protocol-smart-contracts)
- [x] double-check your branch is based on `dev` and targets `dev` 
- [x] Pull request has tests (at least 80% coverage)
- [x] Code is well-commented, linted and follows project conventions
- [x] Documentation is updated (if necessary)
- [x] Description explains the issue/use-case resolved and auto-closes related issues

### CODE REVIEW

File: contracts/AppCoinsIAB.sol [hunk 1]
```diff
@@ -1,7 +1,7 @@
 pragma solidity ^0.4.19;
```
Comment (DioPires) [inline contracts/AppCoinsIAB.sol:1]: #### Tests:
```diff 
+ 104 passing
```
#### Test coverage: 
81.56 %

Comment (DioPires) [inline contracts/AppCoinsIAB.sol:1]: #### Tests:
```diff 
+ 104 passing
```
#### Test coverage: 
81.56 %

Comment (DioPires) [inline contracts/AppCoinsIAB.sol:1]: #### Tests:
```diff 
+ 104 passing
```
#### Test coverage: 
81.56 %

Comment (DioPires) [inline contracts/AppCoinsIAB.sol:1]: #### Tests:
```diff 
+ 104 passing
```
#### Test coverage: 
81.56 %

Comment (DioPires) [inline contracts/AppCoinsIAB.sol:1]: #### Tests:
```diff 
+ 104 passing
```
#### Test coverage: 
81.56 %

Comment (DioPires) [inline contracts/AppCoinsIAB.sol:1]: #### Tests:
```diff 
+ 104 passing
```
#### Test coverage: 
81.56 %

Comment (DioPires) [inline contracts/AppCoinsIAB.sol:1]: #### Tests:
```diff 
+ 104 passing
```
#### Test coverage: 
81.56 %

**Review by GustavoRSSilva — APPROVED** [review]

File: contracts/AppCoinsIAB.sol (hunk 1 above)
Comment (DioPires) [inline contracts/AppCoinsIAB.sol:1]: #### Tests:
```diff 
+ 104 passing
```
#### Test coverage: 
81.56 %

**Pull request merged**